concent_api/utils/logging.py

```diff
@@ -325,6 +334,42 @@ def log_operation_validation_failed(
     )
 
 
+@replace_element_to_unavailable_instead_of_none
+def log_message_received_in_endpoint(
+    logger,
+    application_and_endpoint: str,
+    message_type: str,
+    client_public_key: bytes,
+    content_type: Optional[str] = None,
+    task_id: Optional[str] = None,
+    subtask_id: Optional[str] = None
+
+):
+
+    logger.info(
+        f'A message has been received in `{application_and_endpoint}/` '
+        f'Message type: {message_type} '
+        f'{"TASK_ID:" if task_id is not None else ""} {task_id if task_id is not None else ""}'
+        f'{" SUBTASK_ID:" if subtask_id is not None else ""} {subtask_id if subtask_id is not None else ""}'
+        f'CLIENT_PUBLIC_KEY: {client_public_key} '
+        f'{"Content type:" if content_type is not None else ""} {content_type if content_type is not None else ""} '
+    )
+
+
+def log_json_message(
+    logger,
```

Comment:
This is short signature, there is no need to break it into separate lines.

Reply:
done in all short signatures